documentation/pole-workspace-guide.adoc

```diff
@@ -0,0 +1,419 @@
+WITH point(l) AS corrie
+MATCH (x:Location)<-[:OCCURRED_AT]-(c:Crime)
+WITH x, c, point.distance(point(x), corrie) AS distance
+WHERE distance < 500
+RETURN x.address AS address, count(c) AS crime_total, collect(c.type) AS crime_type, distance
+ORDER BY distance
+LIMIT 10
+----
+
+This is a complex query that pipelines the results from one part of the query to the next.
+The first part of the query matches the `Location` node with the address `1 Coronation Street` and the
+`WITH` clause takes the `point` of that location and assigns it to the variable `corrie` and pipes `corrie` to the next part of the query.
+
+The second `MATCH` clause matches other locations (x) where crimes (c) were committed and then uses the spatial function `point.distance`to calculate the distance between the various other locations and `1 Coronation Street` and limits the distance to less than 500 meters.
+
+The aggregation function `collect()` is used to return a list of values.
+
+[NOTE]
+.Challenge
+====
+Can you find the address farthest away from 1 Coronation St and find out what types of crimes have been committed there?
+
+Keep in mind that you don't want to limit the distance in this query.
+====
+
+[%collapsible]
+.Reveal the solution
+====
+The address farthest away from 1 Coronation St is 156 The Beacons and the crime types are "Violence and sexual offences" and "Drugs".
+This is a query to find this information:
+[source,cypher]
+----
+MATCH (l:Location {address: '1 Coronation Street'})
+WITH point(l) AS corrie
+MATCH (x:Location)<-[:OCCURRED_AT]-(c:Crime)
+WITH x, c, point.distance(point(x), corrie) AS distance
+RETURN x.address AS address, count(c) AS crime_total, collect(c.type) AS crime_type, distance
+ORDER BY distance DESC
+LIMIT 1
+----
+====
+
+In the next section, you will look at people in the graph.
+
+== Find people
+
+There are two different types of people in the graph, `Person` and `Officer`.
+A person can be party to a crime or related to (in one or more hops) a person party to a crime and a crime can be investigated by an officer.
+
+The following query shows what one Inspector Morse is working on:
+
+.Crimes investigated by Inspector Morse
+[source,cypher]
+----
+MATCH (o:Officer {surname: 'Morse'})<-[i:INVESTIGATED_BY]-(c:Crime)
+RETURN *
+----
+
+[NOTE]
+.Challenge
+====
+Write a Cypher query to find a different officer named Devy Larive and the crimes he's _currently_ investigating.
+Find out which type of crime is most common among his open cases.
+
+Hint: The `Crime` nodes have a property called `last_outcome` and you are interested in the ones that are `Under investigation`.
+====
+
+[%collapsible]
+.Reveal the solution
+====
+[source,cypher]
+----
+MATCH (c:Crime {last_outcome: 'Under investigation'})-[i:INVESTIGATED_BY]->(o:Officer {surname: 'Larive'})
+RETURN c.type AS type, count(c) AS number
+ORDER BY count(c) DESC
+----
+
+This should tell you that Officer Larive is currently three Drug crimes
```

Comment:
is currently investigating three Drug crimes

Reply:
Good eye!